spec/vcr/FernService/_create_fern/creates_a_fern.yml

```diff
@@ -0,0 +1,41 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: http://localhost:5000/api/v1/users/110920554030325122207/ferns
+    body:
+      encoding: UTF-8
+      string: name=The+Big+Pepperoni&preferred_contact_method=text&shelf=Family
+    headers:
+      Fern-Key:
+      - FErnCultForLyfe
```

Comment:
do we need to be keeping this hidden? AKA should the vcr folder not be here?

Reply:
Yes, this Fern-Key should not be on GH imo.

Reply:
Should be hidden with the latest push now :)